test/fit_tests/common/fit_common.py

```diff
@@ -703,18 +707,27 @@ def apply_obm_settings_new():
                             nodestatus[node]['status'] = "succeeded"
                         if state in ["failed", "cancelled", "timeout"]:
                             nodestatus[node]['status'] = "pending"
-        if VERBOSITY > 4:
+                            # if the workflow left an invalid OBM, delete it
+                            result = rackhdapi("/api/2.0/nodes/" + node)
+                            if result['status'] == 200:
+                                if result['json']['obms']:
+                                    for ref in result['json']['obms']:
+                                        obmref = ref.get('ref')
+                                        if obmref:
+                                            rackhdapi(obmref, action="delete")
+
+        if VERBOSITY >= 4:
             print "**** Node(s) OBM status:\n", json.dumps(nodestatus, sort_keys=True, indent=4,)
         if "pending" not in str(nodestatus) and "running" not in str(nodestatus):
             # All OBM settings successful
             return True
-        time.sleep(10)
+        time.sleep(30)
```

Comment:
I am fine with the 30 second update, as the max time it takes will be 15 minutes, as we will exit early if all complete successfully.
I'm confused on the comment regarding SEL poller routines and how it relates to the OBM workflows taking longer to run, as the workflow to create IPMI settings doesn't start pollers, but pollers rely on these to be set correctly.  We should look at using the SKU pack options to set the OBMs instead of workflows, as that would speed this up ( I have tested it out a bit and seems to help.)